units/modprobe@.service

```diff
@@ -0,0 +1,17 @@
+#  SPDX-License-Identifier: LGPL-2.1+
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Load kernel module %i
+Documentation=man:modprobe(8)
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
```

Comment:
any reason you added this line? Given that "modprobe -abq" is idempotent and there's nothing we define in ExecStop= I see no reason to keep this unit up, let's hence drop this line.

Reply:
Yeah OK that makes sense, thanks. I thought that having it up would be interesting to see, but it's probably not indeed.

I'm pushing with that dropped now.